updates/35/updates.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO assets(identifier, name, type) VALUES ('eip155:1/erc20:0x5D409e56D886231aDAf00c8775665AD0f9897b56', 'Compound USDS', 'C');INSERT INTO common_asset_details (identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES ('eip155:1/erc20:0x5D409e56D886231aDAf00c8775665AD0f9897b56', 'cUSDSv3', NULL, NULL, NULL, 1729195427, NULL);INSERT INTO evm_tokens (identifier, token_kind, chain, address, decimals, protocol) VALUES ('eip155:1/erc20:0x5D409e56D886231aDAf00c8775665AD0f9897b56', 'A', 1, '0x5D409e56D886231aDAf00c8775665AD0f9897b56', 18, 'compound-v3');INSERT INTO underlying_tokens_list(identifier, weight, parent_token_entry) VALUES('eip155:1/erc20:0xdC035D45d973E3EC169d2276DDab16f1e407384F', '1.0', 'eip155:1/erc20:0x5D409e56D886231aDAf00c8775665AD0f9897b56');
```

Comment:
we always leave a space after `;` in the instructions. Not sure if it is because of the regex in the app or it will work. I would assume that it doesn't affect but would be nice to double check

Reply:
it should work. We can check by playing in develop